packages/cfd/src/Containers/props.types.ts

```diff
@@ -61,3 +62,64 @@ export type TCFDResetPasswordModal = {
     setCFDPasswordResetModal: (value: boolean) => void;
     history: History;
 };
+
+export type TFormValues = {
+    old_password: string;
+    new_password: string;
+    password_type: string;
+};
+
+export type TMultiStepRefProps = {
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+export type TCountdownComponent = {
+    count_from: number;
+    onTimeout: () => void;
+};
+
+export type TCFDPasswordReset = {
+    sendVerifyEmail: () => Promise<VerifyEmailResponse>;
+    account_type: string;
+    account_group: 'real' | 'demo';
+    server: string;
+    password_type: string;
+};
+
+export type TCFDPasswordManagerTabContentWrapper = {
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    steps: Array<{ component: JSX.Element }>;
+};
+
+export type TCFDPasswordManagerTabContent = {
+    toggleModal: () => void;
+    selected_login: string;
+    email: string;
+    setPasswordType: (value: string) => void;
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
```

Comment:
@hirad-rewok Instead of undefined just use '?':
```suggestion
    multi_step_ref?: React.MutableRefObject<TMultiStepRefProps>;
```

Reply:
The difference is that if you put `?` you make your type React.MutableRefObject<...> | undefined but we just need to assign undefined to our MutableRefObject not the whole multi_step_ref

Reply:
i see